### PR TITLE
Unary setup

### PR DESCRIPTION
**Pull request by pmcgrath17**
Adding unary interface for Oak remote attestation noninteractive.

### CODE REVIEW

File: oak_remote_attestation_noninteractive/proto/v1/BUILD [hunk 1]
```diff
@@ -34,6 +34,12 @@ proto_library(
     deps = [":messages_proto"],
 )
 
+proto_library(
+    name = "service_unary_proto",
+    srcs = ["service_unary.proto"],
```
Comment (ipetr0v) [inline oak_remote_attestation_noninteractive/proto/v1/BUILD:39]: Is `service_unary.proto` already in the repo?

Comment (pmcgrath17) [reply oak_remote_attestation_noninteractive/proto/v1/BUILD:39]: I don't believe so. There is only service_streaming.proto for noninteractive remote attestation.

Comment (pmcgrath17) [reply oak_remote_attestation_noninteractive/proto/v1/BUILD:39]: Looks like it didn't get picked up in my commit. Adding now

**Comment by pmcgrath17** [comment]
I have added the proto file to the commit. ptal

**Review by ipetr0v — APPROVED** [review]
Thanks!

**Pull request merged**